.github/workflows/conformance.yml

```diff
@@ -0,0 +1,35 @@
+name: Conformance Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  conformance:
+    permissions:
+      # Needed to access the workflow's OIDC identity.
+      id-token: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@d171c3b028d844f2bf14e9fdec0c58114451e4bf
+
+      - name: Set up JDK 17
```

Comment:
we also support java11, can we matrix this (I hope staging can keep up)?
https://github.com/sigstore/sigstore-java/blob/main/.github/workflows/ci.yaml#L31

Reply:
Yep, having it be a matrix should be fine!

Reply:
[510a16e](https://github.com/sigstore/sigstore-java/pull/245/commits/510a16e6783fc39eef1e61aeb259bef02aa7c7ec)